### PR TITLE
Improve the talk submission form

### PR DESCRIPTION
**Pull request by nbraud**
Based on feedback on the wafer install for DebConf 18.

### CODE REVIEW

File: wafer/talks/models.py [hunk 1]
```diff
@@ -129,7 +129,8 @@ class Meta:
 
     notes = models.TextField(
         null=True, blank=True,
-        help_text=_("Any notes for the conference organisers?"))
+        help_text=_("Any notes for the conference organisers?  "
+                    "Attendees cannot read those."))
```
Comment (stefanor) [inline wafer/talks/models.py]: The "those" seems a bit distant. How about: These are not visible to attendees or the public.

File: wafer/static/css/wafer.css [hunk 2]
```diff
@@ -47,6 +47,10 @@
     border: 5px dashed #000;
 }
 
+.help-block {
+    color: #d9534f;
```
Comment (stefanor) [inline wafer/static/css/wafer.css]: That's quite red. Is that not going to be too close to the colour of errors?

Comment (nbraud) [reply wafer/static/css/wafer.css]: I picked it because it was the color of errors: `.help-block` is set on the validation error messages.

Comment (nbraud) [reply wafer/static/css/wafer.css]: Yeap, confirmed (and removed it from the patch).

Comment (stefanor) [reply wafer/static/css/wafer.css]: err it still seems to be here.

File: wafer/talks/models.py [hunk 3]
```diff
@@ -129,7 +129,8 @@ class Meta:
 
     notes = models.TextField(
         null=True, blank=True,
-        help_text=_("Any notes for the conference organisers?"))
+        help_text=_("Any notes for the conference organisers?  "
+                    "Those are not visible to the public."))
```
Comment (drnlm) [inline wafer/talks/models.py]: I don't like the use of "Those" here - I would prefer "These are not visible ..."

Comment (nbraud) [reply wafer/talks/models.py]: Fixed.

**Pull request merged**